Dependency `colors` may not be `devDependencies`

# Issue guidelines

If you want to add a really specific behavior to the reporter, please consider to [build and use your own display processor](https://github.com/bcaudan/jasmine-spec-reporter/blob/master/docs/customize-output.md).

Please fill in this template, delete this intro and any unrelated text :smile:

## Bug report

Check with "x":

- [x] I have this issue with the latest release of [jasmine-spec-reporter](https://github.com/bcaudan/jasmine-spec-reporter/releases) - 5.0.0
- [x] I have this issue with the latest release of [jasmine](https://github.com/jasmine/jasmine/releases)  - 3.5.0
- [ ] I can reproduce this issue with one of the [examples](https://github.com/bcaudan/jasmine-spec-reporter/tree/master/examples)

**Current behavior**

I have migrate to `4.2.1` to `5.0.0`, following error is occured.
I think  `colors` is not `devDependencies`. It may `dependencies`.
I tried `npm install -D colors` in my project, error is fixed.

```
> jasmine

internal/modules/cjs/loader.js:964
    throw err;
    ^

Error: Cannot find module 'colors'
Require stack:
- D:\src\node_modules\jasmine-spec-reporter\built\display\colors-display.js
- D:\src\node_modules\jasmine-spec-reporter\built\display\execution-display.js
- D:\src\node_modules\jasmine-spec-reporter\built\spec-reporter.js
- D:\src\node_modules\jasmine-spec-reporter\built\main.js
- D:\src\spec\helpers\reporter.js
- D:\src\node_modules\jasmine\lib\jasmine.js
- D:\src\node_modules\jasmine\bin\jasmine.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:961:17)
    at Function.Module._load (internal/modules/cjs/loader.js:854:27)
    at Module.require (internal/modules/cjs/loader.js:1023:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (D:\src\node_modules\jasmine-spec-reporter\built\display\colors-display.js:3:14)
    at Module._compile (internal/modules/cjs/loader.js:1128:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1167:10)
    at Module.load (internal/modules/cjs/loader.js:983:32)
    at Function.Module._load (internal/modules/cjs/loader.js:891:14)
    at Module.require (internal/modules/cjs/loader.js:1023:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'D:\\src\\node_modules\\jasmine-spec-reporter\\built\\display\\colors-display.js',
    'D:\\src\\node_modules\\jasmine-spec-reporter\\built\\display\\execution-display.js',
    'D:\\src\\node_modules\\jasmine-spec-reporter\\built\\spec-reporter.js',
    'D:\\src\\node_modules\\jasmine-spec-reporter\\built\\main.js',
    'D:\\src\\spec\\helpers\\reporter.js',
    'D:\\src\\node_modules\\jasmine\\lib\\jasmine.js',
    'D:\\src\\node_modules\\jasmine\\bin\\jasmine.js'
  ]
}
npm ERR! Test failed.  See above for more details.
```

--- a/package.json
+++ b/package.json
@@ -43,12 +43,13 @@
     "url": "https://github.com/bcaudan/jasmine-spec-reporter/issues"
   },
   "homepage": "https://github.com/bcaudan/jasmine-spec-reporter",
-  "dependencies": {},
+  "dependencies": {
+    "colors": "1.4.0"
+  },
   "devDependencies": {
     "@types/jasmine": "3.5.9",
     "@types/node": "13.9.3",
     "codecov": "3.6.5",
-    "colors": "1.4.0",
     "diff": "4.0.2",
     "jasmine": "3.5.0",
     "jasmine-core": "3.5.0",

